chef/cookbooks/crowbar/templates/default/crowbar-reload-nova-after-upgrade.sh.erb

```diff
@@ -27,21 +27,41 @@ if [[ -f $UPGRADEDIR/crowbar-reload-nova-after-upgrade-ok ]] ; then
     exit 0
 fi
 
+# Check the python-nova package to make sure it is at the latest version with
+# no more recent versions in any zypper repositories (this package can be used
+# as a proxy for making sure all Nova service packages are up to date since
+# they are version locked to python-nova):
+
+if zypper lu -a | grep -w python-nova; then
+  echo "There are available update candidates for python-nova. This means that "
+  echo "the nova service packages on this node are not at the latest possible "
+  echo "version. Please update them and restart the upgrade."
+
+  exit 1
+fi
+
 <% if @nova_controller %>
 for service in conductor scheduler novncproxy serialproxy api; do
     fullname="openstack-nova-$service"
-    if systemctl --quiet is-active $fullname 2>/dev/null ; then
-        systemctl restart $fullname
-    fi
+    systemctl start $fullname
```

Comment:
why did you remove the `is-active` check? do we always have all of these services installed? if something is missing then you should not try to start it.

Reply:
`if systemctl cat $fullname &>/dev/null; then ...` seems to do the trick.

Reply:
Done.